docs/device-emulation.md

```diff
@@ -30,33 +30,74 @@ Because we do not want to modify the daemon, we think it makes sense to *load* a
 state over D-Bus. Each phase can be controlled, which makes it easy to view, and edit, the recorded
 emulation data.
 
-For instance, calling `fwupdmgr emulation-tag` would ask the end user to choose a device to start
+## Getting started
+
+The very first thing to do is to turn on the emulation functionality for fwupd.  It's off by default.
+
+    fwupdmgr modify-config fwupd AllowEmulation true
+
+Restart the daemon.
```

Comment:
I don't actually think you need to restart for hotplug devices. I know the client tool says you have to, but I don't think that's true. Maybe it's safer to restart, I dunno. I do think we need to tell the user to `systemctl restart fwupd.service` or whatever the command is now.

Reply:
But if we flip the default to enabled we don't have to tell them anything.